.github/workflows/release.yaml

```diff
@@ -147,7 +147,6 @@ jobs:
     runs-on: ubuntu-latest
     if: needs.build-skip-check.outputs.chart_changed == 'true'
     needs:
-      - publish-image
```

Comment:
The `publish-image` dependency is unnecessary for this step and is preventing the independent publication of the Helm charts.